buzsaki_lab_to_nwb/huszar_hippocampus_dynamics/behaviorinterface.py

```diff
@@ -2,14 +2,62 @@
 
 import numpy as np
 from pynwb.file import NWBFile, TimeIntervals, TimeSeries
-from pynwb.behavior import SpatialSeries, Position, CompassDirection
+from pynwb.behavior import SpatialSeries, Position
 from hdmf.backends.hdf5.h5_utils import H5DataIO
 
 from neuroconv.utils.json_schema import FolderPathType
 from neuroconv.basedatainterface import BaseDataInterface
 from neuroconv.tools.nwb_helpers import get_module
 
 from scipy.io import loadmat as loadmat_scipy
+from pymatreader import read_mat
```

Comment:
Did we add `pymatreader` to the `requirements.txt` for this project?

Reply:
Also how does it differ from `hdfstrorage`, `scipy.io`, or just `h5py`?

Reply:
It is the cleanest reader I have found as it reproduces the behavior of `scipy.io` even in new files without the need for the forking logic that we have on spikeinterface for example.

Reply:
OK, sound good

Did we add it to the requirements though?

Reply:
scoped `requirements.txt` has been latest practice